python/mxnet/recordio.py

```diff
@@ -166,6 +166,19 @@ def write_idx(self, idx, buf):
 
 
 IRHeader = namedtuple('HEADER', ['flag', 'label', 'id', 'id2'])
+"""An alias for HEADER used for storing metadata (e.g. labels) that accompanies a record.
```

Comment:
Right now the sentence runs on a bit. Please change to:

"An alias for HEADER. Used to store metadata (e.g. labels) accompanying a record."